src/Integration/ProfileConflicts/ConflictsManager.cs

```diff
@@ -0,0 +1,187 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ConflictsManager.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using EnvDTE;
+using Microsoft.VisualStudio;
+using SonarLint.VisualStudio.Integration.Binding;
+using SonarLint.VisualStudio.Integration.Persistence;
+using SonarLint.VisualStudio.Integration.Resources;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+
+namespace SonarLint.VisualStudio.Integration.ProfileConflicts
+{
+    /// <summary>
+    /// Conflicts manager for a SonarQube bound solution
+    /// </summary>
+    internal class ConflictsManager : IConflictsManager
```

Comment:
It's not obvious from the class name or doc comment what can be in conflict; RulesetConc